extensions/amp-date-picker/0.1/amp-date-picker.js

```diff
@@ -230,7 +229,7 @@ class AmpDatePicker extends AMP.BaseElement {
    * @return {?Promise<!JsonObject|!Array<JsonObject>>}
    */
   fetchSrcTemplates_() {
-    if (assertHttpsUrl(this.element.getAttribute('src'), this.element)) {
```

Comment:
`assertHttpsUrl` is already called in `fetchedBatchedJsonFor`, so it was redundant here.